ui/de-lib/src/main/java/org/iplantc/de/apps/client/presenter/categories/AppCategoriesPresenterImpl.java

```diff
@@ -164,28 +167,41 @@ public void onFailure(Throwable caught) {
             @Override
             public void onSuccess(List<AppCategory> result) {
                 filterCategories(result);
-                createWorkspaceTab(selectedAppCategory);
-                createHPCTab();
+                addCategoriesToWorkspaceTree(selectedAppCategory);
+                addCategoriesToHPCTree();
+                autoSelectHPCCategory();
```

Comment:
😝 I'm terrible at naming things.  Yes, your suggestion is much more clear, I'll update it